### ISSUE TITLE
Fully utilize card updating

### ISSUE THREAD

**Issue by olivermt** [integrations/microsoft-teams#45]
It is a bit counter-intuitive that updates to a PR etc makes a reply in the thread instead of updating the card. It also means that actions on a PR makes the reply thread VERY noisy, when most PR updates are mostly relevant as they are and not necessarily in the context of all its history.

Github integration on slack feels more 'interactive' due to utilizing updates instead of thread replies and it should be a good candidate for GH to provide a homogenous UX-experience across the chat platforms.

**Comment by ashokirla** [comment]
@olivermt Thanks for your feedback. The main goal of the GitHub integration with teams is to promote collaboration within the teams and increase GitHub engagement. Threading is one of such feature that can improve collaboration where any new updates are added as a reply and it enables team to have conversations around it. This infact reduces noise caused by too many posts which is one of the feedback we received from our slack customers.

As part of Personal app flow, which we are currently working on, the experience will be tailored to focus on real time updates as collaboration is not required here. This should address your feeback.

We will try to keep our functionalities as uniform as possible across various platforms we integrate (Slack, Teams etc.), however, the experiences can be slightly different based on the support provided by the platform and the way our customers use the platform.

**Comment by olivermt** [comment]
Hi, I can dig that reasoning.

Just to be clear, I meant adaptive cards so that the **same card** is updated when something happens.
Maybe commits look nicer as thread comments, but for 'simple' things like a person being assigned it seems to make more sense to just update the existing card is what I meant.

To use thread replies on the card for all messaging is a great idea, flooding the channel on slack made us make separate channels just for the github spam (which is still an improvement on emails, but not very collaborative).

**Comment by ashokirla** [comment]
@olivermt Yes, we only add actions like comments, reviews and open/close/merge as replies to parent cards. 
Updates to assignees, reviewers, labels, checks etc shown inplace on the same card. We dont have separate replies for them.

Please keep sharing your feedback to make our app better. I am happy to setup a call and discuss in detail if you are interested.

**Comment by bradcutshall** [comment]
We are only using the integration for "pulls" right now as a way to automatically create PR cards in a team when one is created. It would be very nice to not have the "Pull request merged by <user>" reply to the thread when the PR is completed, if this could be an option to disable, that would be very nice. This is noisy and since it can quickly bury an open PR that nobody has claimed or completed yet since older PRs that have recently been reviewed will move in front of the PR that was recently created.

Thanks!
